test-common/test-common-runtime/src/main/java/org/apache/kafka/common/test/ClusterInstance.java

```diff
@@ -258,7 +258,52 @@ default Set<GroupProtocol> supportedGroupProtocols() {
     //---------------------------[wait]---------------------------//
 
     default void waitTopicDeletion(String topic) throws InterruptedException {
-        waitForTopic(topic, 0);
+        Collection<KafkaBroker> brokers = aliveBrokers().values();
+        List<TopicPartition> topicPartitions = IntStream.range(0, 1)
```

Comment:
@chia7712 
It seems that we always have `TopicPartition(topic, 0)` here.
Should we change it to `List.of(new TopicPartition(topic, 0))` ?